tests/benchmarks/FCSBenchmarks/CompilerServiceBenchmarks/BackgroundCompilerBenchmarks.fs

```diff
@@ -151,9 +151,6 @@ type NoFileSystemCheckerBenchmark() =
 
     let mutable benchmark : ProjectWorkflowBuilder = Unchecked.defaultof<_>
 
-    [<ParamsAllValues>]
-    member val UseGetSource = true with get,set
-
     [<ParamsAllValues>]
     member val UseChangeNotifications = true with get,set
```

Comment:
Same thing, should probably change this to `useFileSystem`.

Reply:
Renamed to `useInMemoryDocuments`